tools/fork-network/src/cli.rs

```diff
@@ -85,9 +85,43 @@ enum SubCommand {
 
 #[derive(clap::Parser)]
 struct InitCmd {
-    /// If given, the shard layout in this file will be used to generate the forked genesis state
-    #[arg(long)]
-    pub shard_layout_file: Option<PathBuf>,
+    /// Shard layout override option. Available options:
+    /// - no_override: Use the shard layout from the source chain
```

Comment:
nit: AFAIK you can put doc comments on the enum variants and the help command should display the relevant info